contentcuration/contentcuration/frontend/channelEdit/pages/StagingTreePage/index.vue

```diff
@@ -385,11 +392,21 @@
       },
     },
     watch: {
-      nodeId(newNodeId) {
-        this.elevated = false;
-        this.loadAncestors({ id: newNodeId });
-        this.loadChildren({ parent: newNodeId, root_id: this.stagingId });
-      },
+  nodeId(newNodeId) {
+    this.elevated = false;
+    this.isLoadingchild = true;
+
+    Promise.all([
+      this.loadAncestors({ id: newNodeId }),
+      this.loadChildren({ parent: newNodeId, root_id: this.stagingId }),
+      this.loadCurrentChannel({ staging: true }),
+    ]).then(() => {
+      this.isLoadingchild = false;
+      this.loadCurrentChannelStagingDiff();
```

Comment:
Where did this `this.loadCurrentChannelStagingDiff();` call come from? I don't see it in the previous code

Reply:
The additions made were unnecessary . I simply utilized the promise block from the page and unintentionally neglected to remove the lines above. My apologies for the oversight; I have now removed them.